library/core/functions.general.php

```diff
@@ -3013,6 +3013,10 @@ function SafeRedirect($Destination = FALSE, $StatusCode = NULL) {
     */
    function SaveToConfig($Name, $Value = '', $Options = array()) {
       Gdn::Config()->SaveToConfig($Name, $Value, $Options);
+
+      // Flush Zend OPcache after config file is saved.
+      if (function_exists('opcache_reset'))
+         opcache_reset();
```

Comment:
This isn't the right place to put the call. It would be inside `Gdn_Configuration->Save()`. I know we have some calls to `apc_delete_file()` so maybe that's a good place to look.

Also, please use curly braces even with a one line if statement to match our current coding style.